Firestore/core/src/firebase/firestore/remote/grpc_connection.mm

```diff
@@ -135,6 +135,21 @@
                                        worker_queue_, this, observer);
 }
 
+std::unique_ptr<GrpcUnaryCall> GrpcConnection::CreateUnaryCall(
+    absl::string_view rpc_name,
+    const Token &token,
+    const grpc::ByteBuffer &message) {
+  LOG_DEBUG("Creating gRPC unary call");
```

Comment:
For every log message you add, ask yourself: when we ask a customer to enable logging, what is this message going to tell us? If this is the last log message before a failure, will it help us understand what went wrong?

We do need logging for high-value interactions, especially those that are prone to failure, but usually we only need the one entry and it should clearly describe what we're about to do before we do it and whether or not we succeeded once it's done.

If this particular action here is prone to failure, then let's keep it, but try to add information to the log line here that will help us establish what happened given a customer's log file.

Reply:
Removed. I think I got overzealous replicating [Web client](https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/src/platform_node/grpc_connection.ts#L111).